### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

File: WeakAurasOptions/WeakAurasOptions.lua [hunk 1]
```diff
@@ -1589,7 +1589,7 @@ function OptionsPrivate.OpenTriggerTemplate(data, targetId)
     local loaded, reason = LoadAddOn("WeakAurasTemplates");
     if not(loaded) then
       reason = string.lower("|cffff2020" .. _G["ADDON_" .. reason] .. "|r.")
-      WeakAuras.prettyPrint("Templates could not be loaded, the addon is " .. reason);
+      WeakAuras.prettyPrint(string.format(L["Templates could not be loaded, the addon is "], reason));
```
Comment (mrbuds) [inline WeakAurasOptions/WeakAurasOptions.lua:1592]: `%s` missing

**Comment by mrbuds** [comment]
It seems the order of auras after a drag & drop is reversed as shown in this test https://www.twitch.tv/videos/1445255993

**Comment by mrbuds** [comment]
Note that the few multi drag & drops done in this test are sometime reversed, sometime not

**Comment by InfusOnWoW** [comment]
Ah fixed that. They were reversed if you dropped "before" a aura in a group.

**Comment by mrbuds** [comment]
If i drop it at first position it's reversed https://www.twitch.tv/videos/1445314014

**Comment by InfusOnWoW** [comment]
Right another issue. This time I didn't correctly account for whether you dropped on the top or bottom part of a aura....

**Review by mrbuds — APPROVED** [review]

**Pull request merged**